lib/core/src/persist/send.rs

```diff
@@ -116,15 +118,34 @@ impl Persister {
         &self,
         from_state: PaymentState,
         to_state: PaymentState,
+        is_local: Option<bool>,
     ) -> Result<()> {
         let con = self.get_connection()?;
+        let mut where_clauses = vec!["state = :from_state".to_string()];
+        if let Some(is_local) = is_local {
+            where_clauses.push(format!(
+                "(sync_state.is_local = {} OR sync_state.is_local IS NULL)",
```

Comment:
I realized now that `is_local` is not nullable. I think I was the one to start with also checking for a null value, but unless I'm missing something, it seems it isn't needed.

Reply:
There might not be a row for the swap_state, as it's either not arrived yet, or the rt sync is disabled

Reply:
Ahh right, exactly, now I remember. This brings me to another issue: wouldn't a null `is_local` mean it is local?

Reply:
In our case we only want to set the local ones

Reply:
True, but the method interface allows for is_local to be `Some(false)`, and in that case, it could behave unexpectedly.